backend/model/jpca_ead_exporter.rb

```diff
@@ -0,0 +1,38 @@
+# encoding: utf-8
+require 'nokogiri'
+require 'securerandom'
+
+class EADSerializer < ASpaceExport::Serializer
+  serializer_for :ead
+
+  def serialize_rights(data, xml, fragments)
+    data.rights_statements.each do |rts_stmt|
+      xml.userestrict({ id: "aspace_#{rts_stmt['identifier']}", type: rts_stmt['rights_type'] }) {
```

Comment:
Since JPCA will always use "Other", could we also add the following to line 10:

```
, altrender: rts_stmt['other_rights_basis']
```

That will ensure that the "other rights basis" value also gets exported.  It would mean that the 'altrender' attribute will always be exported, even when empty, but that wouldn't invalidate the EAD file, so I'm fine with that bit of inelegance if you are :)